specification/quota/resource-manager/Microsoft.Quota/stable/2021-03-15/examples/getComputeUsages.json

```diff
@@ -0,0 +1,171 @@
+{
+  "parameters": {
+    "scope": "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus",
+    "api-version": "2021-03-15"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "value": [
+          {
+            "properties": {
+              "id": "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/quotaLimits/standardFSv2Family",
+              "limit": 100,
+              "unit": "Count",
+              "currentValue": 0,
+              "isQuotaApplicable": true,
+              "name": {
+                "value": "standardFSv2Family",
+                "localizedValue": "Standard FSv2 Family vCPUs"
+              }
+            }
```

Comment:
Please add 'type', 'name', 'id' at the top level.

Reply:
Same comment as next one.
Compute doesn't provide the fields.
https://docs.microsoft.com/en-us/rest/api/compute/usage/list - Compute doesn't provides name as - https://docs.microsoft.com/en-us/rest/api/compute/usage/list#usagename
Which is included as name
We don't know the Type, until the RP provides us.
https://armwiki.azurewebsites.net/api_contracts/UsagesAPIContract.html?q=usages - Contract is bit different. Please review.
We can create Id, as we discussed earlier, but it it's not pointing to the real Azure resource owned by Compute.
Please suggest?

Reply:
the top level properties are about your resource type not Compute's.

"type": "Microsoft.Quota/quotaLimits",
"name": "standardFSv2Family",

and id is correct. Just move it to the top level.

Reply:
Added -  "id": "subscriptions/00000000-0000-0000-0000-000000000000/providers/Microsoft.Compute/locations/eastus/providers/Microsoft.Quota/quotaLimits/standardFSv2Family",
at top level.